.github/workflows/ci-ubuntu.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - master
       - experimental
+      - 'v*-branch'
```

Comment:
Nitpick more than anything: is there any reason to bless `-branch` as a suffix in the name of version branches, newly introduced by #2439 ? (I didn't catch it there)

It's not as if we have `master-branch` as a name... or is the intention that these names stand out because they'll be (eventually) ephemeral?

Would `v*.*` suffice instead?

Reply:
The problem with `v*.*` is that then we have branches and tags with the same name, and Git complains in certain circumstances because of the ambiguity. 

I'd certainly be happy with `release-v*`, I'll change it :+1: